cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -36,22 +36,22 @@ extensions:
             index_prefix: "jaeger-main"
             spans:
               date_layout: "2006-01-02"
-              rollover_frequency: "day"
+              rollover_frequency: "1day"
```

Comment:
can you create another PR (to merge before this one) that changes the `RolloverFrequency` field type to `time.Duration`

Reply:
Okay!